control/statesp.py

```diff
@@ -1459,7 +1459,7 @@ class LinearICSystem(InterconnectedSystem, StateSpace):
 
     """
 
-    def __init__(self, io_sys, ss_sys=None):
+    def __init__(self, io_sys, ss_sys=None, connection_type=None):
```

Comment:
Is there a reason to allow the connection type to be overridden?  It seems like this should just be inherited from the underlying `InterconnectedSystem`.